Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -50,11 +52,30 @@ class FavoritesViewController: UIViewController {
         $0.autoresizingMask = [.flexibleWidth]
     }
     
-    /// Called after user taps on ddg popup to set it as a default search enginge in private browsing mode.
-    var ddgPrivateSearchCompletionBlock: (() -> Void)?
+    private let ddgLogo = UIImageView(image: #imageLiteral(resourceName: "duckduckgo"))
+    
+    private let ddgLabel = UILabel().then {
+        $0.numberOfLines = 0
+        $0.textColor = BraveUX.GreyD
+        $0.font = UIFont.systemFont(ofSize: 14, weight: UIFont.Weight.regular)
+        $0.text = Strings.DDG_promotion
+    }
+    
+    private lazy var ddgButton = UIControl().then {
```

Comment:
remove `lazy`

Reply:
Can't, it uses `self` in its initialization (adding a target)